packages/scxa-marker-gene-heatmap/src/MarkerGeneHeatmap.js

```diff
@@ -26,12 +26,20 @@ async function addModules () {
 addModules()
 
 const splitPhrase = (phrase, maxLineLength = 12) => {
-  const splitPhrase = [``]
+  const splitPhrase = []
 
   const words = phrase.split(/\s+/)
+
+  // Add first word to the first line if splitPhrase is empty
```

Comment:
At this point `splitPhrase` is always empty. You defined it as empty in line 29 and nothing changes it.
The comment does not reflect what is happening in the code.
It adds the first word to the first line if `words` has any content and not when `splitPhrase` is empty.
Could you please remove this comment?
Thanks